internal/commands/push.go

```diff
@@ -35,8 +35,8 @@ Optionally, specific directory can be passed as a second argument, e.g.:
 
 	$ conftest push instrumenta.azurecr.io/my-registry:v1 path/to/dir
 
-Conftest leverages the ORAS library under the hood. This allows arbitrary artifacts to 
-be stored in compatible OCI registries. Currently open policy agent bundles are supported by 
+Conftest leverages the ORAS library under the hood. This allows arbitrary artifacts to
+be stored in compatible OCI registries. Currently open policy agent bundles are supported by
```

Comment:
nit: It's not related to your change, but please uppercase Open Policy Agent here.